installer/x86_64/install.sh

```diff
@@ -385,6 +385,7 @@ if [ "$install_env" != "sonic" ]; then
     }
 else
     demo_mnt="/host"
+    TAR_EXTRA_OPTION="--numeric-owner"
```

Comment:
onie is using another version of tar which does not support this option, but it works so it must using uid/gid by default.